src/__tests__/internal/LinkButton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import LinkButton from '../../components/internal/LinkButton';
+
+// Shallow Rendering
+// https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md
+describe('LinkButton - Shallow Rendering', () => {
+    it('to have 1 LinkButton class', () => {
+      const wrapper = shallow(
+        <LinkButton
+          id="LinkButton"
+          text="This is the link"
+          className="LinkClass"
+        />,
+      );
+      expect(wrapper.find('.LinkClass')).to.have.length(1);
+    });
+});
+
+  // Static Rendered Markup
+  // https://github.com/airbnb/enzyme/blob/master/docs/api/render.md
+describe('LinkButton - Static Rendered Markup', () => {
```

Comment:
Add similar tests to the `Button`, to test the loading spinner, changing of text, nothing happening when the link is clicked when it is in a loading state and to test that the `onClick` handler works